Tzivi-details.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>name: Tzivi</h1>
```

Comment:
you need to write last Name

Reply:
your note is not smart
but you're so cute!😚